src/Application/View/Extension/DataHelpersExtension.php

```diff
@@ -67,6 +68,13 @@ public function getFilters(): array
         ];
     }
 
+    public function getTests(): array
+    {
+        return [
+            new TwigTest('url_safe', \Toolbox::isUrlSafe(...)),
```

Comment:
I was not aware of this syntax.

https://www.php.net/manual/en/functions.first_class_callable_syntax.php

Reply:
Neither was I until the PHP version in composer changed and my IDE suggested it as a replacement for the older syntax I typed for this line.